packages/medusa/src/utils/__tests__/transaction/transaction-builder.ts

```diff
@@ -0,0 +1,376 @@
+import e from "cors"
+import { TransactionBuilder } from "../../transaction/transaction-builder"
+
+describe("TransactionBuilder", () => {
+  describe("Basic methods", () => {
```

Comment:
todo: you can remove this describe and start with the test directly for this one